src/main/java/com/flightstats/hub/app/HubResponseFilter.java

```diff
@@ -0,0 +1,16 @@
+package com.flightstats.hub.app;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class HubResponseFilter implements ContainerResponseFilter {
+
+    @Override
+    public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) {
+        responseContext.getHeaders().add("Hub-Node", HubHost.getLocalNamePort());
```

Comment:
Curious - how does this get the real name while inside of docker?

Reply:
I believe its a byproduct of Docker's `host` network. If you jump in one of the running Hub containers it thinks it's hostname is that of it's host.

Reply:
We're just calling `InetAddress.getLocalHost().getHostName()` in the application code.

Reply:
There is an open issue for when running this in the "single" Hub:
https://github.com/flightstats/hub/issues/1028